docs/website/layouts/index.html

```diff
@@ -495,6 +492,8 @@ <h2>Declarative Policy</h2>
 	[_, client_id] := split(uri_type_san, "=")
 }
 
+http_request := input.attributes.request.http
```

Comment:
Interesting. Is this a Regal rule? 🤔

Reply:
Indeed, it's related to https://docs.styra.com/regal/rules/imports/avoid-importing-input